### ISSUE TITLE
Whatsapp method of background live location

### ISSUE THREAD

**Issue by nicolasvahidzein** [ekasetiawans/flutter_background_service#431]
I'm sorry to open a ticket for this but I don't seem to understand how whatsapp is able to continually provide a live location update in the background for up to 8 hours and in the read me it says we are limited to only every 15mn.

How is whatsapp able to accomplish this in a way that we cannot?

Thank you so much.

**Comment by ImanGhasemiArani** [comment]
hello, 
se this related [topic](https://stackoverflow.com/questions/55683726/how-to-perform-background-service-while-battery-optimization-is-active),

**Comment by nicolasvahidzein** [comment]
@ImanGhasemiArani thanks but nothing in there really addresses my needs.

**Comment by ekasetiawans** [comment]
Background location is requires special permission, you have to explain the usages when you want to publish your app in appstore or playstore.

**Comment by nicolasvahidzein** [comment]
Yes but that is not the issue here. I said how to code it not the playstore guidelines.

**Comment by reliable-dev786** [comment]
@ekasetiawans I want the same feature like i want to share my current location continuously unless the user stop it or after some hours like 3-4.
I'm using this package but it stop after sometime in terminated state of app.
How can i achieve this scenario? I think the battery optimizer stop the service so how can i prevent it?
